client/asset/doge/doge.go

```diff
@@ -23,6 +25,7 @@ const (
 	defaultFeeRateLimit = 1_000_000
 	minNetworkVersion   = 1140400
 	walletTypeRPC       = "dogecoindRPC"
+	feeConfs            = 10
```

Comment:
10? not 1?  Does it work more reliably with a larger number?

Reply:
Makes sense

Reply:
On a mainnet node running for a while

| blocks |   fee rate |
|--------|-----------:|
| 1      |         -1 |
| 2      |         -1 |
| 3      | 5.46232920 |
| 4      | 0.01003906 |
| 5      | 0.01003906 |
| 6      | 0.01003906 |
| 7      | 0.01003906 |
| 8      | 0.01003906 |
| 9      | 0.01003906 |
| 10     | 0.01003906 |

:shrug:

Reply:
Just got this on a mainnet redeem `2022-04-14 03:47:14.101 [TRC] CORE[doge]: Obtained local estimate for 2-conf fee rate, 537604`.  After that I opened a terminal to check, and  `estimatefee` is saying `5.37140538` for confs 2 through 10 (!)  :facepalm: So turns out `estimatefee` is complete shit.  I'm not sure what we want to do about that, but I'm almost thinking we go with a static fee equal to 3-5x of the "recommended" fee of 0.01 DOGE/kB